packages/frontend/src/components/RefreshDbtButton/index.tsx

```diff
@@ -31,6 +33,39 @@ const RefreshDbtButton: FC<ComponentProps<typeof BigButton>> = (props) => {
     )
         return <div></div>;
 
+    if (
+        health.data?.mode === LightdashMode.CLOUD_BETA &&
```

Comment:
why the cloud beta check?

Reply:
just to ensure it is not triggered locally. our cloud is still on Beta according to the lightdash mode
![image](https://user-images.githubusercontent.com/1983672/195631314-41ac79e8-1514-4dd6-8cca-715300bba0c3.png)

Reply:
yeah, you're right, just tested it locally and it doesn't make sense.